neuro_san/internals/run_context/langchain/core/run_context_runnable.py

```diff
@@ -68,7 +57,7 @@
                                          ])
 
 
-class NeuroSanRunnable(RunnablePassthrough):
+class RunContextRunnable(NeuroSanRunnable):
```

Comment:
Derive from NeuroSanRunnable now.
Some member variables move to that superclass now.